macros/sql_gen/build_metric_sql.sql

```diff
@@ -1,5 +1,6 @@
-{%- macro build_metric_sql(metric_dictionary, grain, dimensions, secondary_calculations, start_date, end_date, calendar_tbl, relevant_periods, calendar_dimensions, dimensions_provided) %}
+{%- macro build_metric_sql(metric_dictionary, metric_config, grain, dimensions, secondary_calculations, start_date, end_date, calendar_tbl, relevant_periods, calendar_dimensions, dimensions_provided) %}
     
+    {%- set default_value_null = metric_config.get("default_value_null") if metric_config else False  -%}
```

Comment:
Do you have to protect against non-boolean inputs here? This isn't something I've ever had to think about because I didn't realise this could be 100% user-defined without any oversight from us. What happens if someone does 

```yml
metrics: 
   name: my_metric
   config: 
     default_value_null: banana
```
?